src/factory/OlStyle.js

```diff
@@ -109,6 +120,56 @@ export const OlStyleFactory = {
     return new Fill({
       color: styleConf.fillColor
     });
+  },
+
+  /**
+   * Creates and returns a basic OL style object for texts.
+   *
+   * @param {Object} labelConf Style config object for labels
+   * @returns {ol/style/Text} The OL style object for texts
+   */
+  getTextStyle (labelConf) {
+    const textConf = { ...labelConf };
```

Comment:
why is this necessary? 
a comment might be helpful

Reply:
It ensures a clone of the config is created, so no in place modification of the config is done. Good point, I'll add a comment.

Reply:
See e32d245